### PR TITLE
Website (homepage/showcase): Update background colors

### PR DESCRIPTION
**Pull request by moutro**
Update Dark mode and light mode to match original designs for:

- UA banner
- homepage subheader
- section containers
- hero banner

Add border-bottom to hero banner, update footer border-top.

cc: @asafkorem

### CODE REVIEW

**Comment by asafkorem** [comment]
Thanks @moutro, that's amazing!! I will review it soon 🤩

File: website/src/components/CustomBanner/StandWithUkraine.module.css [hunk 1]
```diff
@@ -2,7 +2,7 @@
   display: flex;
   justify-content: center;
   align-items: center;
-  background-color: #1f243c;
+  background-color: #1f253c;
```
Comment (asafkorem) [inline website/src/components/CustomBanner/StandWithUkraine.module.css:5]: Did they also ask to change this color?

File: website/src/css/custom.css [hunk 2]
```diff
@@ -44,13 +44,13 @@ html[data-theme='dark'] .header-github-link:before {
   border-top: 1px solid #f2eeeee8;
 }
 
-html[data-theme='dark'] {
-  --ifm-background-color: #202429;
+html[data-theme='dark']:root {
+  --ifm-background-color: #12161b;
```
Comment (asafkorem) [inline website/src/css/custom.css:48]: Why do we need the `:root` selector here?

**Review by asafkorem — APPROVED** [review]
Looks good to me! 🙌🏼 
@moutro please answer these two questions before I merge the PR

File: website/src/css/custom.css (hunk 2 above)
Comment (noomorph) [reply website/src/css/custom.css:48]: @asafkorem is right. Just use either html or :root

Comment (noomorph) [reply website/src/css/custom.css:48]: Btw, you can merge with line 17.

File: website/src/components/CustomBanner/StandWithUkraine.module.css (hunk 1 above)
Comment (moutro) [reply website/src/components/CustomBanner/StandWithUkraine.module.css:5]: They included it in the figma image for style matching. When inspecting, this is the hex showing - very minor but it does seem to be just a bit darker IMO. I can revert it if you prefer the previous. Please let me know!

Comment (asafkorem) [reply website/src/components/CustomBanner/StandWithUkraine.module.css:5]: Dont revert, that's great :+1:

File: website/src/css/custom.css (hunk 2 above)
Comment (asafkorem) [reply website/src/css/custom.css:48]: `:root` has higher specification level than `html`, so there might be something that overrides the `html` version.. 🤔

Comment (asafkorem) [reply website/src/css/custom.css:48]: Actually maybe `html` selector doesn't work with JSX, because JSX is JavaScript **XML**..

Comment (moutro) [reply website/src/css/custom.css:48]: I'm not 100% sure but I think an `html` selector is not directly relevant to JSX and agree with what you said above.

Comment (moutro) [reply website/src/css/custom.css]: My apologies for the extra commit - had a technical issue and pushed 2nd commit without removing commented code. Fixed in latest commit.

**Pull request merged**